refinery/templates/core/all_files.html

```diff
@@ -0,0 +1,251 @@
+{% extends "base.html" %}
+
+{% load humanize %}
+{% load markup %}
+{% load static from staticfiles %}
+
+
+{% block head_html %}
+  <link rel="stylesheet" href="{% static "vendor/spectrum/spectrum.css" %}" />
+  <link rel="stylesheet" href="{% static "styles/provenance-visualization.css" %}" />
+  <link rel="stylesheet" href="{% static "styles/file-browser.css" %}" />
+  <link rel="stylesheet" href="{% static "vendor/tipsy/src/stylesheets/tipsy.css" %}" />
+  <link rel="stylesheet" href="{% static "vendor/select2/select2.css" %}" />
+  <link rel="styleSheet" href="{% static "vendor/angular-ui-grid/ui-grid.min.css" %}"/>
+  <link rel="stylesheet" href="{% static "vendor/ui-select/dist/select.min.css" %}" />
+{% endblock head_html %}
+
+<!-- DIFF: Title -->
+{% block title %} - All Files{% endblock %}
+
+{% block subheader %}
+  <div class="page-header">
+    <!-- DIFF: Title -->
+    <h1>All Files</h1>
+  </div>
+{% endblock %}
+
+{% block content %}
+  <!-- tab headers -->
+
+<div ng-controller="DataSetNavCtrl" ng-cloak>
```

Comment:
Not Required: DataSetNavCtrl - required for the file tabbing, DataSetUiModeCtrl - required for analyses state

Create your own controller for the all-files-view